Website/src/rendering/src/assets/css/base/sitecore.css

```diff
@@ -12,4 +12,8 @@
   .scpm {
     display: none !important;
   }
+
+  .experience-editor-active {
```

Comment:
For some reason, in my portal instance, this directive is not bundled in the final CSS file and the EE shows a white 122px gap between the header and the main section.
https://jel-pull0076-cm.sitecoredemo.com/?sc_mode=edit
I will continue to troubleshoot in a couple of hours.

Reply:
This was caused by Tailwindcss purging unused definitions in production builds. We were missing some source files in the purge configuration.